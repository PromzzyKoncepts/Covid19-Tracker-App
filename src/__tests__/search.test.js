import React from "react";
import renderer from "react-test-renderer";
import { Provider } from "react-redux";
import store from "../redux/Store";
import Search from "../components/search";
it("renders Component correctly", () => {
  const tree = renderer.create(
    <Provider store={store}>
      <Search />
    </Provider>
  );
  expect(tree).toMatchSnapshot();
});
