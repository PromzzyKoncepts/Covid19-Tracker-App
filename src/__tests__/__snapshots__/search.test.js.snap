// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Component correctly 1`] = `
<div
  className="search"
>
  <div
    className="center input-group input-group-lg"
    onChange={[Function]}
  >
    <span
      className="input-group-text"
      id="inputGroup-sizing-lg"
    >
      Search
    </span>
    <input
      aria-describedby="inputGroup-sizing-sm"
      aria-label="Large"
      className="form-control"
      placeholder="Country name"
    />
  </div>
</div>
`;
